src/ast/parse.lalrpop

```diff
@@ -101,9 +101,12 @@ Schedule: Schedule = {
     <ident:Ident> => Schedule::Run(RunConfig { ruleset: ident, until: None }),
 }
 
-Cost: Option<usize> = {
-    ":cost" <UNum> => Some(<>),
-    => None,
+Cost: CostFn = {
+    ":cost" <UNum> => CostFn::Constant(<>),
+    ":cost" <Ident> => CostFn::Table(false, <>),
```

Comment:
Perhaps it would be simpler to force the user to declare the table themself, then typecheck it properly? This is pretty good too but may be confusing.